src/Microsoft.DocAsCode.Build.RestApi/ApplyOverwriteDocumentForRestApi.cs

```diff
@@ -47,6 +47,21 @@ public class ApplyOverwriteDocumentForRestApi : ApplyOverwriteDocument
                 return ((RestApiRootItemViewModel)articleModel.Content).Children.Where(c => c.Uid == uid);
             }));
 
+        public Func<FileModel, string, IHostService, IEnumerable<RestApiTagViewModel>> GetTagsFromOverwriteDocument =
+            (((overwriteModel, uid, host) =>
+            {
+                return OverwriteDocumentReader.Transform<RestApiTagViewModel>(
+                    overwriteModel,
+                    uid,
+                    s => BuildRestApiDocument.BuildTag(host, s, overwriteModel, content => content != null && content.Trim() == Constants.ContentPlaceholder));
+            }));
+
+        public Func<FileModel, string, IHostService, IEnumerable<RestApiTagViewModel>> GetTagItemsToOverwrite =
+            (((articleModel, uid, host) =>
+            {
+                return ((RestApiRootItemViewModel)articleModel.Content).Tags.Where(c => c.Uid == uid);
+            }));
```

Comment:
good catch!

---
In reply to: [65476613](https://github.com/dotnet/docfx/pull/346#discussion_r65476613) [](ancestors = 65476613)